docs/demo/materials/02-atoms/buttons/00-button-flat.html

```diff
@@ -0,0 +1,13 @@
+---
+name: button flat
```

Comment:
Just talked to @xonic and he agrees flat is not a great name as it carries a bit of the flat design connotation. I like btn-link because it's a button that looks like a link (and is even created by importing link properties)